### PR TITLE
KOGITO 2884 - loadmore redesign

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

This pr contains changes that redesigns the loadmore component.

JRIA:  https://issues.redhat.com/browse/KOGITO-2884

### CODE REVIEW

File: ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx [hunk 1]
```diff
@@ -28,26 +34,68 @@ const LoadMore: React.FC<IOwnProps & OUIAProps> = ({
...
+    const selectedValue = parseInt(event.currentTarget.id, 10);
+    setLoadMoreValue(selectedValue);
   };
 
+  const dropdownItems = [
+    <DropdownItem key="loadmore10" component="button" id="10">
+      <Split hasGutter>
+        <SplitItem>Load 10 more</SplitItem>
+        {loadMoreValue === 10 && (
+          <SplitItem>
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: can we introduce a function? so that we don't repeat the same JSX 4 times.

Comment (jstastny-cz) [reply ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: Maybe a function for whole DropdownItem, that would take arguments `count: number` ? sth like (types to be added)
```
const dropdownItem = (count: number) => {
  return <DropdownItem key={'loadmore'+count} component="button" id={count}>
      <Split hasGutter>
        <SplitItem>Load {count} more</SplitItem>
        {loadMoreValue === count && (
          <SplitItem>
            <CheckIcon size="sm" color="var(--pf-global--info-color--100)" />
          </SplitItem>
        )}
      </Split>
    </DropdownItem>
}
```

**Review by jstastny-cz — CHANGES_REQUESTED** [review]

**Comment by jstastny-cz** [comment]
Typo in the PR title.

File: ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx (hunk 1 above)
Comment (AjayJagan) [reply ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: Hi @jstastny-cz - I have converted the JSX to a function . Please have a look

**Comment by AjayJagan** [comment]
> Typo in the PR title.

corrected the typo in PR title :). Thanks.

**Review by nmirasch — APPROVED** [review]
Looks good to me, ready to merge once PR gets the green, thanks Ajay!

**Review by jstastny-cz — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

**Review by pefernan — CHANGES_REQUESTED** [review]
@AjayJagan good job, I like prefer this desing to the previous one by far, 

but nitpicking Pere here... there's an issue when clicking on "Load 10 more" button, when the button is focused it adds a border that with the spinner makes it dance a little (see snapshot), it's not too much but it affects my TOC ;-)

image

IDK how this could be arranged (maybe bluring the dropdown or removing the border...) however, I know this is a small issue so consider merging and fix it later if you want.

**Review by pefernan — APPROVED** [review]
Agree to merge, we can improve it with https://issues.redhat.com/browse/KOGITO-3253

**Pull request merged**